parsec/api/data/entry.py

```diff
@@ -37,8 +37,12 @@ def new(cls: Type[EntryIDTypeVar]) -> EntryIDTypeVar:
 class EntryName(str):
     __slots__ = ()
 
-    def __init__(self, raw: str):
+    def __new__(cls, raw: str) -> "EntryName":
+        import unicodedata
```

Comment:
import should be on top of the file (import is costly, even when the module has already been imported)

```
$ python -m timeit -s "pass" "import unicodedata"
10000000 loops, best of 3: 0.0796 usec per loop
$ python -m timeit -s "import unicodedata" "pass"
100000000 loops, best of 3: 0.00513 usec per loop
```